content/docs/engine-dev/architecture/prefabs/_index.md

```diff
@@ -0,0 +1,645 @@
+---
+linkTitle: Prefab System 
+title: Developer guide to the Prefab system in O3DE
```

Comment:
I think lowercase 'prefab' should be used.

https://www.o3de.org/docs/contributing/to-docs/style-guide/format/

Reply:
The guide actually says :
```
Use standard capitalization, not sentence capitalization, for both the metadata title (title), the table-of-contents title (linkTitle).
```
So, leaving it as is.